### ISSUE TITLE
Clearify the project status

### ISSUE THREAD

**Issue by buhtz** [keithbowes/user-agent-switcher#11]
Could you please clearify the status of your project/fork and the "original" one.

It looks like that chrispedericks version is out-dated. But it is still available on the Firefox AddOn page. It should be removed from there. And in chris's repository should be a big notification message about that this repo is out-dated. But he doesn't allow issues there.

Your project looks fresh. ;) It would be nice if you could point more specific to the older ones and say: "They are outdated." or something like that. And  it would be nice to know that is the difference between your version and the old one.

**Comment by keithbowes** [comment]
Yeah. I've been procrastinating for SeaMonkey did get support for WebExtensions before porting this over, but I might start working on it in Firefox.  I can't get my version of this extension in AMO, because it changes user-agent preferences.  Hopefully, the future WebExtensions version that changes the headers and navigator object on individual web requests without changing preferences will be approved.
